careers/careers/templates/careers/internships.jinja

```diff
@@ -0,0 +1,228 @@
+{% extends "base.jinja" %}
+
+{% block page_title %}Experience an internship at Mozilla{% endblock %}
+
+{% block meta_description %}Interns at Mozilla work alongside industry leaders on meaningful projects like online
+privacy protection.{% endblock %}
+
+{% block body_class %}internships-page{% endblock %}
+
+{% block site_css %}
+{{ super() }}
+<link href="{{ static('css/internships.css') }}" rel="stylesheet" />
+{% endblock %}
+
+{% block content %}
+
+<section class="mzp-c-hero mzp-t-dark">
+  <div class="mzp-l-content">
+    <div class="mzp-c-hero-body">
+      <h1 class="mzp-c-hero-title">Make an Impact</h1>
+      <div class="mzp-c-hero-desc">
+        <p>Ready to work on meaningful projects that will reshape privacy and safety on the internet? Mozilla is the
+          right spot for you.</p>
+      </div>
+      <p class="mzp-c-hero-cta">
+        <a class="mzp-c-button mzp-t-dark" href="{{ url('careers.listings')|urlparams(position_type='Intern') }}">Apply
+          for an internship</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="l-internships-section">
+  <div class="mzp-l-content c-section-heading">
+    <h2>What to Expect</h2>
+    <p>Besides working alongside the brightest and kindest minds— here’s what else you’ll do.</p>
+  </div>
+  <div class="mzp-l-content mzp-l-card-half">
+    <div class="mzp-c-card mzp-c-card-medium">
```

Comment:
I think instead of doubling up on standard cards, where one has no text and the other has no image, these blocks could each be a feature card. It seems like the only customization it would need is a different font-size for the title and `align-items: start;` to override the vertical centering (assuming the top alignment is required and they can't just be centered anyway, as is standard).

But that said I know none of this is using "proper" Protocol so this does work as a quick hack and we can refactor later. My biggest issue with the current card layout is the funky ordering when things stack on mobile, but I can live with that for the sake of getting it live ASAP.